### ISSUE TITLE
Can't get globalOptions to work for HTTP Server

### ISSUE THREAD

**Issue by maggiecelentano95** [highcharts/node-export-server#92]
I am trying to send a C# POST request to the HTTP Server:

string result = client.UploadString(new Uri("http://localhost:7801/"), "POST", "{\"infile\":" + serializedChart + ",\"b64\":1, \"globalOptions\":" + globalSettings}")

This has been working perfectly fine until I added the globalOptions parameter, with globalSettings being:

            var gs = new  { lang = new { thousandsSep = "," } };
            var globalSettings = Newtonsoft.Json.JsonConvert.SerializeObject(gs);

I have also tried:
            var globalSettings = "{\"lang\": {\"thousandsSep\":\",\"}}";

Neither of these cases have worked. I am receiving the error:
             phantom worker 2 unexpected data - SyntaxError: Expected token ']'

I have also tried:

            var globalSettings= "\"{lang:{thousandSep:','}}\"";

This didn't cause an error, but the thousands separator on the graph was still a space instead of a comma. 

Please let me know how I can properly set globalOptions on a POST request.

**Comment by marouanehassani** [comment]
+1 I am having the same issue when setting globalOptionsn the server fails with 

```
unexpected data - SyntaxError: Expected token ']'
  undefined:1 in appendChild
  :16
  :19
```
. I cannot find any workaround to get globalOptions to work with httpServer

**Comment by cvasseng** [comment]
Fixed in 48e2421 / `2.0.10`.
